sips/networking/last_change_round.md

```diff
@@ -0,0 +1,99 @@
+| Author     | Title                                             | Category | Status |
+|------------|---------------------------------------------------|----------|--------|
+| Niv Muroch | Handle non committee change round for all signers | Core     | Draft  |
```

Comment:
why not add decided msgs as well? isn't it the same thing in terms of spec?

Reply:
decided msg includes more than 1 signer. so no need to save by signers it is enough as is